.github/workflows/github-pages.yml

```
@@ -1,5 +1,11 @@
 on:
   workflow_dispatch:
+    inputs:
+      opt-in-to-sparse-registry:
+        description: Whether the environment variable CARGO_UNSTABLE_SPARSE_REGISTRY is set
```

Consider this wording:
```suggestion
        description: Enable/disable CARGO_UNSTABLE_SPARSE_REGISTRY (https://blog.rust-lang.org/2022/06/22/sparse-registry-testing.html`
```